public/app/listeners/exportSpace.js

```diff
@@ -33,76 +34,111 @@ const validateConditions = (
   return spaceCondition && (studentCondition || visiblityCondition);
 };
 
+const prepareArchive = async (
+  archive,
+  db,
+  { space, spaceId, selection, userId }
+) => {
+  const {
+    space: isSpaceSelected,
+    actions: isActionsSelected,
+    resources: isResourcesSelected,
+  } = selection;
+
+  const isStudent = db.get('user.settings.studentMode').value();
+
+  // export the user's resources, private and public
+  const resources = isResourcesSelected
+    ? db
+        .get(APP_INSTANCE_RESOURCES_COLLECTION)
+        .filter((resource) =>
+          validateConditions(resource, spaceId, userId, isStudent)
+        )
+        .value()
+    : [];
+
+  const actions = isActionsSelected
+    ? db
+        .get(ACTIONS_COLLECTION)
+        .filter((action) =>
+          validateConditions(action, spaceId, userId, isStudent)
+        )
+        .value()
+    : [];
+
+  // copy prepackaged apps in space
+  if (isSpaceSelected) {
+    // get prepackaged apps
+    const isPrepackagedApp = (item) =>
+      item?.category === APPLICATION && item?.asset?.startsWith(VAR_FOLDER);
+
+    // eslint-disable-next-line no-restricted-syntax
+    for (const [phaseIdx, phase] of space.phases.entries()) {
+      const { items = [] } = phase;
+      // eslint-disable-next-line no-restricted-syntax
+      for (const [itemIdx, item] of items.entries()) {
+        if (isPrepackagedApp(item)) {
+          const { asset, name } = item;
+          const assetFolder = path.dirname(asset);
+          const mainFilename = path.basename(asset);
+
+          // archive prepackaged app
+          archive.directory(assetFolder, name);
+
+          // change asset in json
+          // eslint-disable-next-line no-param-reassign
+          space.phases[phaseIdx].items[itemIdx].asset = path.join(
```

Comment:
Does this give the relative path? Try exporting from one computer and importing in another, just in case.

Reply:
Or does import handle absolute paths fine? I forget.

Reply:
This gives a relative path from the space folder but I will test using another computer.

Reply:
Looks good to me now. Just check if import/export works with different (computer) users / computers.

Reply:
It worked on my windows computer. I'll add some tricks for the tests to pass and I will merge.